dockerfiles/Dockerfile-for-frpc

```diff
@@ -1,11 +1,11 @@
-FROM alpine:3.12.0 AS temp
+FROM alpine:3.15.0 AS temp
```

Comment:
Can we just use `alpine:3` here and no need to always update it.

Reply:
@fatedier  yes `alpine:3` also works updating it

Reply:
@fatedier  updated, please merge and build when possible 👍